packages/stack/contracts-manager/src/index.js

```diff
@@ -325,7 +325,7 @@ export default class ContractsManager {
           contract.swarmHash = compiledContract.swarmHash;
           contract.gasEstimates = compiledContract.gasEstimates;
           contract.functionHashes = compiledContract.functionHashes;
-          contract.abiDefinition = compiledContract.abiDefinition;
+          contract.abiDefinition = contractConfig && contractConfig.abiDefinition ? contractConfig.abiDefinition : compiledContract.abiDefinition;
```

Comment:
If we don't need to test whether `contractConfig` is truthy (there are places in the code above where it's not checked), then would it be simpler as:

```suggestion
          contract.abiDefinition = contractConfig.abiDefinition ?? compiledContract.abiDefinition;
```

Reply:
Could combine both above suggestions of optional chaining and null coalescing 🤣, so that we can be sure we're null-checking `contractConfig` (if we need to of course, as @michaelsbradleyjr pointed out).
```suggestion
          contract.abiDefinition = contractConfig?.abiDefinition ?? compiledContract.abiDefinition;
```